storage/inmemory/src/lib.rs

```diff
@@ -27,6 +37,22 @@ impl backend::ReadOps for StorageMaps {
     }
 }
 
+impl<'i> backend::PrefixIter<'i> for StorageMaps {
+    // TODO: Define a custom iterator type and do this without boxing
```

Comment:
This TODO is probably not relevant anymore?

Reply:
Yes, outdated TODO. Thanks for pointing it out